app/kumactl/data/install/k8s/metrics/prometheus/server.yaml

```diff
@@ -435,16 +435,16 @@ spec:
               mountPath: /etc/config
               readOnly: true
         - name: prometheus-server
-          image: "prom/prometheus:v2.35.0"
+          image: "prom/prometheus:v3.1.0"
           imagePullPolicy: "IfNotPresent"
           args:
-            - --storage.tsdb.retention.time=15d
             - --config.file=/etc/config/prometheus.yml
             - --storage.tsdb.path=/data
+            - --storage.tsdb.retention.time=15d
+            - --storage.tsdb.retention.size=7GB
```

Comment:
is this 7GB based on something or something specific to our setup?

Reply:
it didn't change, it was just moved to group together similar configurations, and it' snot based on anything specific, just values we picked some time ago